### ISSUE TITLE
Test request not showing the variable for baseURL

### ISSUE THREAD

**Issue by deepanshu-rajput-git** [scalar/scalar#4158]
### What happens?

We have configured the variable `workspace` for the `baseURL` of server:

`openapi: 3.0.0`
`servers:`
`  - description: NeetoCal APIs`
`    url: https://{workspace}.neetocal.com/api/external/v1`
`    variables:`
`      workspace:`
`        default: spinkart   # This is the default value`
`        description: Your workspace's subdomain`

Here is my [OpenAPI spec file](https://sandbox.scalar.com/e/q5df_)

### What did you expect to happen?

Editing the workspace is not getting reflected in `Test request`:

https://deepanshu-bigbinary.neetorecord.com/watch/f363e4e1-385f-4d4f-9967-29b5af699323

**Comment by antlio** [comment]
hey @deepanshu-rajput-git thank you for reporting this! a PR has been open regarding this.

**Comment by deepanshu-rajput-git** [comment]
@antlio, please keep me updated on the release.

**Comment by antlio** [comment]
hey @deepanshu-rajput-git, the fix has been deployed yesterday, can you let me know how everything is running now for you?

**Comment by deepanshu-rajput-git** [comment]
@antlio : No, its not working for me.
![image](https://github.com/user-attachments/assets/f0025bba-8804-4d8f-ae08-b909f4a65533)

Here is the link to my [API docs](https://apidocs.neetocal.com/apis#tag/bookings/GET/bookings)

**Comment by antlio** [comment]
@deepanshu-rajput-git sorry i mixed things up with another of your issue (early morning!). the PR is under review, i'll keep you updated when it's deployed!

**Comment by deepanshu-rajput-git** [comment]
@antlio: Are the updates deployed?

**Comment by antlio** [comment]
hey @deepanshu-rajput-git, yes it should be, do you still have the issue?

**Comment by deepanshu-rajput-git** [comment]
Hey, @antlio any updates on this?

**Comment by deepanshu-rajput-git** [comment]
@antlio: Thanks I will take a look to that. And what about hiding `Download OpenAPI Document` button from the webpage and about the baseURL was not getting reflected in the Test request modal.

**Comment by antlio** [comment]
can you try adding `"hideDownloadButton": true` to your config panel?

<img width="1002" alt="image" src="https://github.com/user-attachments/assets/6f17027a-cc65-49e8-9791-d1f0ad97107d" />

**Comment by deepanshu-rajput-git** [comment]
Yes, that's working fine in the editor but I have my GiHub project and it's not working fine there. Please take a look:
<img width="834" alt="Screenshot 2024-12-18 at 3 00 37 PM" src="https://github.com/user-attachments/assets/e1c82ea9-06f8-4e4b-beae-a37506178a3a" />

**Comment by antlio** [comment]
@deepanshu-rajput-git you can find here [an example of our api reference](https://github.com/scalar/scalar/blob/main/packages/api-reference/index.html) with a commented configuration.

i don't have issue on my end, do you see a difference:
```js
    <script>
      var configuration = {
        hideDownloadButton: true,
      }

      document.getElementById('api-reference').dataset.configuration =
        JSON.stringify(configuration)
    </script>
```

**Comment by deepanshu-rajput-git** [comment]
@antlio : Please confirm, this is supposed to be added in the scalar.config.json of the GitHub Project for my Scalar API docs.

`{
        hideDownloadButton: true,
      }`

**Comment by antlio** [comment]
hey @deepanshu-rajput-git - sorry i'm not sure what you are implying by GitHub Project here, can you tell me more about what do you use here to generate the reference?

**Comment by deepanshu-rajput-git** [comment]
@antlio: Here is the record: https://deepanshu-bigbinary.neetorecord.com/watch/603c6b55-7f61-4f6c-84f2-5c6f018a77c9

**Comment by deepanshu-rajput-git** [comment]
@antlio: Please take this up as a priority; we have been waiting for a long.

**Comment by antlio** [comment]
@deepanshu-rajput-git i request some more infos internally, for the time being you should be able to do add the following style:

```css
.download-button {
  display: none !important
}
```

as a quick fix in order to not being blocked by that, will follow up with more infos regarding the config as soon as i can

**Comment by antlio** [comment]
@deepanshu-rajput-git for the time being you should be able to do add the following style - as we did for the dark mode toggle - :

```css
.download-button {
  display: none !important
}
```

as of right now we do not support reference config that way, however will get back to you as soon as this changes

**Comment by antlio** [comment]
Could be added same as we did [here for the dark mode toggle](https://github.com/scalar/scalar/issues/4158#issuecomment-2550759674), let me know how this goes

**Comment by deepanshu-rajput-git** [comment]
But we don't have any option to customize the GitHub projects using the Webpage.
Ref: 
https://deepanshu-bigbinary.neetorecord.com/watch/6bedb9af-7f95-40a9-8184-ea857f937adc

**Comment by deepanshu-rajput-git** [comment]
> hey @deepanshu-rajput-git, not yet on the docs.scalar.com, will make sure it's done today and hit you up here!

@antlio : Hey, is this fix deployed, I am still not able to edit the `workspace` variable in `Test request` modal.

**Comment by antlio** [comment]
hey @deepanshu-rajput-git, sorry about that, i believe it should be, do you have the variable input showing up in your client modal:

<img width="905" alt="image" src="https://github.com/user-attachments/assets/22b55c87-5bfd-447b-abd2-f051d072135b" />

**Comment by deepanshu-rajput-git** [comment]
No, @antlio it's not there.
![image](https://github.com/user-attachments/assets/949d723e-7780-4bb6-9386-704df3fcb0a4)

**Comment by deepanshu-rajput-git** [comment]
@antlio : Thanks for the fix but the header is not passed in the API.
Ref: https://deepanshu-bigbinary.neetorecord.com/watch/c7e68eb0-b2c9-4199-8536-8b362873ccae

**Comment by antlio** [comment]
@deepanshu-rajput-git sorry about that, do you mind opening another issue regarding that specific issue? will give it a look as soon as the new year start!

**Comment by deepanshu-rajput-git** [comment]
- @antlio : I filed a separate issue for the same: https://github.com/scalar/scalar/issues/4354
